yoyodyne/models/pointer_generator.py

```diff
@@ -401,10 +401,16 @@ def forward(
             last_hiddens = self.init_hiddens(len(batch))
         if not self.has_features_encoder:
             if self.beam_width is not None and self.beam_width > 1:
-                # predictions = self.beam_decode(
-                # batch_size, x_mask, encoder_out, beam_width=self.beam_width
+                # predictions, scores = self.beam_decode(
+                #     encoder_out=source_encoded,
+                #     mask=batch.source.mask,
+                #     beam_width=self.beam_width,
+                #     n=self.n
                 # )
-                raise NotImplementedError
+
+                # LSTM beam search does not work with pointer generator LSTM.
+                raise NotImplementedError(
+                    f"Beam search not implemented for {self.name} model.")
```

Comment:
Same formatting note about close paren.

Also, don't we inherit this from from the base class?

Reply:
I made a similar comment somewhere else; we might as well just call `self.beam_decode` (that way you document what its arguments ought to be) and then let the base class throw the exception.